llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -3522,7 +3565,7 @@ class VPlan {
 
   /// Contains all the external definitions created for this VPlan. External
   /// definitions are VPValues that hold a pointer to their underlying IR.
-  SmallVector<VPValue *, 16> VPLiveInsToFree;
+  SmallVector<VPValue *, 16> VPLiveIns;
```

Comment:
Seems redundant to hold all live-ins in both VPLiveIns SmallVector and above Value2VPValue map?
The latter could be renamed `Value2VPLiveIn`, although `VPLiveIn`'s are actually `VPValue`'s.